src/qt/forms/overviewpage.ui

```diff
@@ -638,66 +839,38 @@
                        <property name="spacing">
                         <number>12</number>
                        </property>
-                       <item row="4" column="0">
+                       <item row="1" column="0">
                         <widget class="QLabel" name="labelzBalanceMatureText">
                          <property name="toolTip">
                           <string>Mature: more than 20 confirmation and more than 3 mints of the same denomination after it was minted.
 These zPIV are spendable.</string>
                          </property>
                          <property name="text">
-                          <string>Mature:</string>
+                          <string>Available:</string>
                          </property>
                         </widget>
                        </item>
                        <item row="1" column="1">
-                        <widget class="QLabel" name="labelzBalance">
+                        <widget class="QLabel" name="labelzBalanceMature">
                          <property name="font">
                           <font>
                            <weight>75</weight>
                            <bold>true</bold>
                           </font>
                          </property>
-                         <property name="cursor">
-                          <cursorShape>IBeamCursor</cursorShape>
-                         </property>
                          <property name="toolTip">
-                          <string>All available zPIV, unconfirmed and immature zPIV included.</string>
+                          <string>Mature: more than 20 confirmation and more than 3 mints of the same denomination after it was minted.
+These zPIV are spendable.</string>
                          </property>
                          <property name="text">
-                          <string notr="true">0.000 000 00 PIV</string>
+                          <string>0.000 000 00 PIV</string>
```

Comment:
should keep the `notr="true"` here

Reply:
Same. It was not voluntarily removed. I just moved parts around. Will add.